app/views/miq_policy/_form.html.haml

```diff
@@ -27,10 +27,10 @@
             = _("Applies To")
           .col-md-8
             - if @policy.id
-              = MiqPolicy::POLICY_TOWHAT_APPLIES_TO_CLASSES[@edit[:new][:towhat]]
+              = MiqPolicy.POLICY_TOWHAT_APPLIES_TO_CLASSES(@edit[:new][:towhat])
```

Comment:
As discussed in the other PR, now that this is a method we should use underscore case

```suggestion
              = MiqPolicy.policy_towhat_applies_to_classes(@edit[:new][:towhat])
```